Gemfile.lock

```diff
@@ -180,28 +193,31 @@ GEM
     parser (3.1.0.0)
       ast (~> 2.4.1)
     path_expander (1.1.0)
-    racc (1.5.2)
+    public_suffix (4.0.6)
+    puma (5.6.2)
```

Comment:
I'm intrigued to see how this plays with the `gem 'puma', '~> 3.7'` requirement for `puma_min`.